site/en/docs/privacy-sandbox/user-agent/index.md

```diff
@@ -181,7 +179,7 @@ increasing in complexity.
 
 Scaled availability for User-Agent reduction means the fully reduced UA string
 shipped on all Chrome devices. Reduction is planned to begin with a Chrome
```

Comment:
Reduction is planned to begin with a ...
Is this still correct?

Reply:
Should be "Reduction began with..."